wpilibc/src/generate/main/native/cpp/hid.cpp.jinja

```diff
@@ -89,4 +90,18 @@ bool {{ ConsoleName }}Controller::GetTouchpadPressed() {
 bool {{ ConsoleName }}Controller::GetTouchpadReleased() {
   return GetRawButtonReleased(Button::kTouchpad);
 }
-{% endif %}
+{%- endif %}
+
+void {{ ConsoleName }}Controller::InitSendable(wpi::SendableBuilder& builder) {
+  builder.SetSmartDashboardType("HID");
+  builder.AddStringProperty("ControllerType", [] { return "{{ ConsoleName }}"; }, nullptr);
+{%- for trigger in triggers %}
+  builder.AddDoubleProperty("{{ capitalize_first(trigger.name) }}", [=, this] { return Get{{ capitalize_first(trigger.name) }}Axis(); }, nullptr);
```

Comment:
`[this]` is enough for the lambda capture list, as there are no implicit captures. Applies to the others as well.